### PR TITLE
achievements page layout created

### PR DESCRIPTION
**Pull request by OleksandrHavron**

### CODE REVIEW

File: front-end/src/Components/pages/achievementsPage/AchievementCard.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		const { card } = this.props
+		return (
+
+			<Card className="achievementCard">
+				<Card.Body>
+					<div className="achievementCardTitle">
+						<div className="achievementIcon">
+							<Badge  />
+						</div>
+						<Card.Title className="achievementTitle">
```
Comment (vvelizariy) [inline front-end/src/Components/pages/achievementsPage/AchievementCard.js]: Needs reformat, tabulation is actually too big, 8 characters, should be 4 maximum.
Also blank lines are required, only when they are used for separating logical blocks, or independent components.

File: front-end/src/Components/pages/achievementsPage/AchievementsPage.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    render() {
+        const { achievements } = this.state
+        return (
+            <>
+                <div className="achievementsWrapper">
+                    <h1 className="text-center">Achievements</h1>
+                    <div className="HeadContainer">
+                        <div className="HeadInnerContainer">
+
```
Comment (vvelizariy) [inline front-end/src/Components/pages/achievementsPage/AchievementsPage.js]: What is whis blank ine for?

File: front-end/src/Components/pages/achievementsPage/AchievementsPage.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                        </div>
+                    </div>
+                    {achievements.map(card => {
```
Comment (vvelizariy) [inline front-end/src/Components/pages/achievementsPage/AchievementsPage.js]: May be shortened
`{achievements.map(card => <AchievementCard card={card} />);}`

File: front-end/src/Components/pages/achievementsPage/AchievementsPage.js (hunk 2 above)
Comment (OleksandrHavron) [reply front-end/src/Components/pages/achievementsPage/AchievementsPage.js]: I forget to delete this lines. I used it for align heading.

**Review by vvelizariy — APPROVED** [review]

**Review by MetalMari — APPROVED** [review]

**Review by Maksson — APPROVED** [review]

**Pull request merged**